GoogleChromeLabs/ps-analysis-tool
# Fix: Handle Infinite loading of library detection section on 404 page.

## Description

It was observed that when you visit a 404 page, the library detection section goes into infinite loop. This PR aims to handle that case by showing a error text.

## Relevant Technical Choices

- Use `chrome.webNavigation.onErrorOccurred.addListener` to know the error occured.

## Testing Instructions

- Visit any website like linkedin.com and then change the url to something invalid like exampledddfdfdfdd.com
- Library detection section should now show a error message.

## Additional Information:



## Screenshot/Screencast

![image](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/6297436/e6723024-2096-4bbb-9cb2-6f2090b5bdf7)


---

## Checklist



- ~ This code is covered by unit tests to verify that it works as intended.~



Fixes #
